src/theme/web/theme.js

```diff
@@ -37,6 +37,13 @@ theme.color.greyLight = '#dbdbdb';
 theme.color.grey = '#aeaeae';
 theme.color.greyDark = '#727171';
 theme.color.greyDarker = '#4a4a4a';
+theme.color.jade = '#03B778';
+theme.color.jadeDark = '#108B65';
```

Comment:
Also what about `jadeDarker`?

Reply:
According to design `darker` shade is not being used . Also making jade as primary color will be a leaf-ui refactor when all the components are designed according to rebranding